lib/modules/platform/gitlab/index.ts

```diff
@@ -1121,12 +1121,14 @@ export async function getIssue(
   useCache = true,
 ): Promise<Issue | null> {
   try {
-    const opts: GitlabHttpOptions = { memCache: false };
-    /* v8 ignore next: temporary code */
+    const opts: GitlabHttpOptions = {};
+    /* v8 ignore start: temporary code */
     if (useCache) {
       opts.cacheProvider = memCacheProvider;
       delete opts.memCache;
```

Comment:
```suggestion

```

Reply:
Yeah, it's better to rewrite for clarity